spec/acceptance/windows_spec.rb

```diff
@@ -1,37 +1,55 @@
 # run a test task
 require 'spec_helper_acceptance'
+require 'beaker-task_helper/inventory'
+require 'bolt_spec/run'
 
 describe 'windows package task', if: fact('osfamily') == 'windows' do
+  include Beaker::TaskHelper::Inventory
+  include BoltSpec::Run
+
+  let(:module_path) { RSpec.configuration.module_path }
+  let(:config) { { 'modulepath' => module_path } }
+  let(:inventory) { hosts_to_inventory }
+
+  def run(params)
+    run_task('package::windows', 'default', params, config: config, inventory: inventory)
+  end
+
   package_to_use = 'notepadplusplus.install'
   before(:all) do
-    hosts.each do |host|
-      on(host, 'cmd.exe /c puppet module install puppetlabs-chocolatey')
-      apply_manifest('include chocolatey')
-    end
+    on(default, 'cmd.exe /c puppet module install puppetlabs-chocolatey')
+    apply_manifest_on(default, 'include chocolatey')
   end
+
   describe 'install action' do
     it "install #{package_to_use}" do
-      apply_manifest("package { \"#{package_to_use}\": ensure => absent, }")
-      result = run_task(task_name: 'package::windows', params: "action=install name=#{package_to_use}")
-      expect_multiple_regexes(result: result, regexes: [%r{install}, %r{success}, %r{Ran on 1 node}])
+      apply_manifest_on(default, "package { \"#{package_to_use}\": ensure => absent, }")
+      result = run('action' => 'install', 'name' => package_to_use)
+      expect(result[0]['status']).to eq('success')
+      expect(result[0]['result']['action']).to match(%r{install})
     end
   end
+
   describe 'uninstall action' do
     it "uninstall #{package_to_use}" do
-      apply_manifest("package { \"#{package_to_use}\": ensure => present, }")
-      result = run_task(task_name: 'package::windows', params: "action=uninstall name=#{package_to_use}")
-      expect_multiple_regexes(result: result, regexes: [%r{install}, %r{success}, %r{Ran on 1 node}])
+      apply_manifest_on(default, "package { \"#{package_to_use}\": ensure => present, }")
+      result = run('action' => 'uninstall', 'name' => package_to_use)
+      expect(result[0]['status']).to eq('success')
+      expect(result[0]['result']['action']).to match(%r{uninstall})
```

Comment:
@MikaelSmith Is this what we expect the structure of the `package::windows` task to be? I ask because it appears to be different than the linux task and needed to be updated to pass Jenkins CI. 

Windows:
```
    24:   describe 'install action' do
    25:     it "install #{package_to_use}" do
    26:       apply_manifest_on(default, "package { \"#{package_to_use}\": ensure => absent, }")
    27:       result = run('action' => 'install', 'name' => package_to_use)
    28:       require 'pry'; binding.pry
 => 29:       expect(result[0]['status']).to eq('success')
    30:       expect(result[0]['result']['action']).to match(%r{install})
    31:     end
    32:   end
    33: 
    34:   describe 'uninstall action' do

[1] pry(#<RSpec::ExampleGroups::WindowsPackageTask::InstallAction>)> result
=> [{"node"=>"xca8yxo2w8ppx0w.delivery.puppetlabs.net",
  "status"=>"success",
  "result"=>
   {"status"=>"success",
    "name"=>"notepadplusplus.install",
    "action"=>"install"}}]
```
linux
```
    28:     it 'uninstall rsyslog' do
    29:       apply_manifest_on(default, "package { 'rsyslog': ensure => present, }")
    30:       result = run('action' => 'uninstall', 'name' => 'rsyslog')
    31:       expect(result[0]['status']).to eq('success')
    32:       require 'pry'; binding.pry
 => 33:       expect(result[0]['result']['status']).to match(%r{install})
    34:     end
    35:   end
    36: 
    37:   describe 'upgrade', if: (fact('operatingsystem') == 'CentOS' && fact('operatingsystemmajrelease') == '7') do
    38:     it 'upgrade httpd' do

[1] pry(#<RSpec::ExampleGroups::LinuxPackageTask::UninstallAction>)> result
=> [{"node"=>"tjloscdgqw7s81j.delivery.puppetlabs.net",
  "status"=>"success",
  "result"=>{"status"=>"rsyslog uninstall"}}]
```

Reply:
It was different. I didn't try to change that.

Reply:
I don't really like either, Windows is probably a bit better. I was going to leave improving them as separate work.